src/openassetio-python/tests/bridge/CMakeLists.txt

```diff
@@ -118,3 +119,26 @@ openassetio_add_test_fixture_dependencies(
     openassetio.internal.python-bridge-test
     openassetio.internal.install
 )
+
+#-----------------------------------------------------------------------
+# Create CTest target for tests that intentionally exclude _openassetio
```

Comment:
[fix] Can we explain why?

Reply:
Sure can.

b872947d3c3c394c8eb97005fa7093af3ac269b5